adal/src/main/java/com/microsoft/aad/adal/AuthenticationResult.java

```diff
@@ -440,13 +444,13 @@ final void setFamilyClientId(final String familyClientId) {
         mFamilyClientId = familyClientId;
     }
 
-    final void setAuthority(final String authority) {
+    public final void setAuthority(final String authority) {
```

Comment:
This doesn't look like it needs to change; I see usages inside of `OAuth2` and nowhere else... can this be left at default visibility?

Reply:
this method is called in BrokerClient in Ad-accounts, so I need to make it public as it was in Unity-fork